### PR TITLE
i18n Improvements

### PR DESCRIPTION
**Pull request by ramiy**

### CODE REVIEW

File: inc/template-tags.php [hunk 1]
```diff
@@ -85,7 +91,13 @@ function twentyseventeen_entry_footer() {
 	if ( 'post' === get_post_type() ) {
 		$tags_list = get_the_tag_list( '', $separate_meta );
 		if ( $tags_list ) {
-			printf( '<span class="tags-links">' . __( 'Tagged %1$s', 'twentyseventeen' ) . '</span>', $tags_list ); // WPCS: XSS OK.
+			echo '<span class="tags-links">';
+			printf(
+				/* translators: %s: Tags list */
+				__( 'Tagged %s', 'twentyseventeen' ),
+				$tags_list
+			); // WPCS: XSS OK.
```
Comment (grappler) [inline inc/template-tags.php]: The comment `// WPCS: XSS OK.` needs to be on the same line as `$tags_list`

File: front-page.php [hunk 2]
```diff
@@ -62,7 +62,7 @@
 					wp_reset_postdata();
 				else :
 					// output placeholder anchor
-					echo '<article class="panel-placeholder panel twentyseventeen-panel twentyseventeen-panel' . esc_attr( $twentyseventeencounter ) .'" id="panel' . esc_attr( $twentyseventeencounter ) . '"><span class="twentyseventeen-panel-title">' . sprintf( __( 'Panel %1$s Placeholder', 'twentyseventeen' ), esc_attr( $twentyseventeencounter ) ) . '</span></article>';
+					echo '<article class="panel-placeholder panel twentyseventeen-panel twentyseventeen-panel' . esc_attr( $twentyseventeencounter ) .'" id="panel' . esc_attr( $twentyseventeencounter ) . '"><span class="twentyseventeen-panel-title">' . sprintf( __( 'Panel %s', 'twentyseventeen' ), esc_attr( $twentyseventeencounter ) ) . '</span></article>';
```
Comment (grappler) [inline front-page.php]: I don't think you want to remove the word "Placeholder"

File: inc/template-tags.php [hunk 3]
```diff
@@ -31,20 +31,26 @@ function twentyseventeen_posted_on() {
...
 	);
 
 	// Check to make sure we have more than one category before writing to page.
 	$categories_list = get_the_category_list( $separate_meta );
 	if ( $categories_list && twentyseventeen_categorized_blog() ) {
-		$categories = sprintf( _x( 'in %1$s', 'prefaces list of categories assigned to the post', 'twentyseventeen' ), $categories_list ); // WPCS: XSS OK.
+		$categories = sprintf(
+			/* translators: %s: A list of categories assigned to the post */
+			__( 'in %s', 'twentyseventeen' ),
+			$categories_list
```
Comment (grappler) [inline inc/template-tags.php]: The comment `// WPCS: XSS OK.` needs to be on the same line as `$categories_list`

File: comments.php [hunk 4]
```diff
@@ -28,7 +28,8 @@
 		<h2 class="comments-title">
 			<?php
 				printf( // WPCS: XSS OK.
-					_nx( 'One thought on &ldquo;%2$s&rdquo;', '%1$s thoughts on &ldquo;%2$s&rdquo;', get_comments_number(), 'comments title', 'twentyseventeen' ),
+					/* translators: 1: comments count 2: Post title */
+					_n( 'One thought on &ldquo;%2$s&rdquo;', '%1$s thoughts on &ldquo;%2$s&rdquo;', get_comments_number(), 'twentyseventeen' ),
```
Comment (grappler) [inline comments.php]: This section should done like in the `_s` PR. https://github.com/Automattic/_s/pull/1012/files

**Review by grappler — CHANGES_REQUESTED** [review]

File: inc/template-tags.php (hunk 3 above)
Comment (ramiy) [reply inc/template-tags.php]: done.

File: inc/template-tags.php (hunk 1 above)
Comment (ramiy) [reply inc/template-tags.php]: done.

File: front-page.php (hunk 2 above)
Comment (ramiy) [reply front-page.php]: Reverted.

File: comments.php (hunk 4 above)
Comment (ramiy) [reply comments.php]: Separate PR.

Comment (grappler) [reply comments.php]: I am not sure what you mean.

**Comment by melchoyce** [comment]
Hey @ramiy, do you know if you'll have time in the next couple of days to make updates to this PR? Just want to get an idea of where this is at. :)

**Comment by ramiy** [comment]
Seems like most of my suggestion implemented. Closing the PR.

**Pull request closed without merging**